src/DynamoCoreWpf/Commands/AutomationSettings.cs

```diff
@@ -116,33 +116,6 @@ internal bool CanSaveRecordedCommands
             }
         }
 
-        /// <summary>
-        /// When HomeWorkspace.RunSettings.RunType is set to Automatically, "HomeWorkspace"
-        /// starts its internal "DispatcherTimer" whenever its content is being 
-        /// modified. This timer starts a round of evaluation after a predefined 
-        /// amount of time has ellapsed, preventing modifications in quick 
-        /// succession from triggering too many evaluations. However, the timer 
-        /// does not always have a chance to tick. This is especially true when 
-        /// AutomationSettings exhausted all available commands in its list and 
-        /// is ready to end the current test run. The shutdown timer that  
-        /// AutomationSettings kicks start may tick before the evaluation timer 
-        /// in WorkspaceModel has a chance to tick. When this happens, validation
-        /// code at the end of the recorded test ends up with invalid evaluation
-        /// results, failing the test case.
-        /// </summary>
-        /// <returns>Returns true if there is a pending evaluation and that the 
-        /// shutdown process should be deferred.</returns>
-        /// 
-        private bool HasPendingEvaluation
-        {
-            get
-            {
-                return
-                    owningDynamoModel.Workspaces.OfType<HomeWorkspaceModel>()
-                        .Any(hw => hw.IsEvaluationPending);
```

Comment:
`HomeWorkspaceModel` simply returned a literal `false` for `IsEvaluationPending` - this simply removes this useless method.